.github/workflows/review.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  review:
+    name: Review
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          version: 7
+          run_install: false
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        shell: bash
+        run: |
+          echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        name: Setup pnpm cache
+        with:
+          path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
```

Comment:
While this is good practice, I think for a project of this small size and only 12 dependencies, it's not worth it setting up a whole cache for dependencies. The run itself with cache is about 45 seconds, which is very short. All the extra steps to setting up the cache, downloading it and uploading it may even not make a difference in the end.

I did a quick test without cache and the whole job took only 41 seconds: https://github.com/redux-utilities/redux-actions/actions/runs/4009885719/jobs/6885759017

So in my opinion we could remove this as we're not winning anything here, and even if, we're talking about few seconds, but we're still way under 1 minute, which is fast.